src/main/kotlin/org/ionproject/integration/extractor/implementation/ITextPdfExtractor.kt

```diff
@@ -0,0 +1,42 @@
+package org.ionproject.integration.extractor.implementation
+
+import com.itextpdf.kernel.pdf.PdfDocument
+import com.itextpdf.kernel.pdf.PdfReader
+import com.itextpdf.kernel.pdf.canvas.parser.PdfTextExtractor
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import org.ionproject.integration.extractor.`interface`.PdfExtractor
+import org.ionproject.integration.extractor.exception.PdfExtractorException
+import org.ionproject.integration.utils.Try
+
+class ITextPdfExtractor : PdfExtractor {
+    /**
+     * Extract text data from pdf file locate at [pdfPath]
+     * @return Try<MutableList<String>>
+     *     Success - String list contains all extracted data in text format
+     *     Failure - PdfExtractorException
```

Comment:
We shouldn't need to add these as comments. It should be explicit on the types. Perhaps we need to encode the exception type on the `Try`.

Reply:
The returned type should document what are the returned options. Unfortunately, the `Try` design allows that for the success case but not for the error case.